server/static/partials/admin/assignment.detail.html

```diff
@@ -3,6 +3,13 @@
 
   <div ng-controller='GroupController' class="box">
   <tabset justified="true">
+    <tab heading="Submissions">
+      <div class='tab-wrapper'>
```

Comment:
Here's the interface that Albert requested:

- For an assignment, all submissions should be placed in the autograding queue
- A button to purge the queue
- A button to add all FinalSubmissions into the queue
- A button for each individual submission to add it manually into the queue so that it will be re-autograded